pkgs/desktops/deepin/setup-hook.sh

```diff
@@ -0,0 +1,33 @@
+# Helper functions for deepin packaging
```

Comment:
Maybe we should tell Deepin about a thing called `PATH`? :rofl: 
Stuff like hardcoding `/bin/ls` shouldn't be needed.

Reply:
There are plenty of binaries (and other files) that are hardcoded like that on DDE. It is going to give some work to have all of them fixed for NixOS.

Reply:
There are at least two ways of fixing some of the hard coded paths for binaries:
- fix its path (`/usr/bin/dde-file-manager` becomes `${dde-file-manager}/bin/dde-file-manager` in `deepin-screenshot`, for instance); it may result in an additional run time dependency
- remove the directory, leaving only the file name (`/usr/bin/dde-file-manager` becomes `dde-file-manager`); it will work only if the directory is in `$PATH`, and the dependency may have to be explicitly installed

Some days ago someone told me on IRC that the first approach is better. But I am not sure of that, because sometimes the dependency may be optional and costly.

Any opinions?